### PR TITLE
RFE: add test for signal and ptrace OBJ_PID records

### PR DESCRIPTION
**Pull request by rgbriggs**
See: https://github.com/linux-audit/audit-testsuite/issues/81

Signed-off-by: Richard Guy Briggs <rgb@redhat.com>

### CODE REVIEW

File: tests/signal/test [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+);
+ok( $result, 0 );    # Was an event found?
+
+# test if we generate the obj_pid records correctly
+my $line;
+my $found_signal = 0;
+my $found_ptrace = 0;
+while ( $line = <$fh_out> ) {
+
+    # find the CONFIG_CHANGE record
```
Comment (pcmoore) [inline tests/signal/test:125]: Comment doesn't match the code.

File: tests/signal/test [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    if ( $line =~ /^type=OBJ_PID / ) {
+
+        # find the lost value
```
Comment (pcmoore) [inline tests/signal/test:128]: Same problem, not a match for the code.

**Comment by rgbriggs** [comment]
Ok, updated, removed comments since the code is pretty self-explanatory.

**Comment by pcmoore** [comment]
Merged via 1e2c58c5c2e0f3c756f1a7f02895b9d33ebda300, thanks.

**Pull request closed without merging**